graphql/batch_executor_test.go

```diff
@@ -275,17 +275,18 @@ func TestNonExpensiveExecution(t *testing.T) {
 }
 
 type counterGoroutineScheduler struct {
-	wg sync.WaitGroup
-
+	wg    sync.WaitGroup
 	count int64
 }
 
 func (q *counterGoroutineScheduler) Run(resolver graphql.UnitResolver, initialUnits ...*graphql.WorkUnit) {
 	q.runEnqueue(resolver, initialUnits...)
-
 	q.wg.Wait()
 }
 
+func (q *counterGoroutineScheduler) RunAll(ctx context.Context, executionUnits []*graphql.ExecutionUnit) {
+}
```

Comment:
Maybe a comment here explaining that this does nothing because the test doesn't need it?